src/main/scala/no/ndla/audioapi/AudioApiProperties.scala

```diff
@@ -40,7 +40,7 @@ object AudioApiProperties extends LazyLogging {
   val SearchServer = propOrElse("SEARCH_SERVER", "http://search-audio-api.ndla-local")
   val SearchRegion = propOrElse("SEARCH_REGION", "eu-central-1")
   val RunWithSignedSearchRequests = propOrElse("RUN_WITH_SIGNED_SEARCH_REQUESTS", "true").toBoolean
-  val SearchIndex = "audios"
+  lazy val SearchIndex = propOrElse("SEARCH_ALIAS", "audios")
```

Comment:
Men er det sånn at det som er hovedproblemet ditt her @runesto at du vil fjerne:
``` setEnv("SEARCH_ALIAS", "integration-test-index")``` i ```UniteSuite``` til fordel for å bare gjøre slik i ```SearchServiceTest```:
```    indexService.createIndexWithName("integration-test-index")
```
istedet for slik det er nå som er:
```
    indexService.createIndexWithName(ArticleApiProperties.SearchIndex)
```